packages/flutter/lib/src/material/user_accounts_drawer_header.dart

```diff
@@ -67,62 +71,78 @@ class _AccountDetails extends StatelessWidget {
   final VoidCallback onTap;
   final bool isOpen;
 
-  Widget addDropdownIcon(Widget line) {
-    final Widget icon = new Icon(
-      isOpen ? Icons.arrow_drop_up : Icons.arrow_drop_down,
-      color: Colors.white
-    );
-    return new Expanded(
-      child: new Row(
-        crossAxisAlignment: CrossAxisAlignment.center,
-        children: line == null ? <Widget>[icon] : <Widget>[
-          new Expanded(child: line),
-          icon,
-        ],
-      ),
-    );
-  }
-
   @override
   Widget build(BuildContext context) {
     final ThemeData theme = Theme.of(context);
-    Widget accountNameLine = accountName == null ? null : new DefaultTextStyle(
+    final MaterialLocalizations localizations = MaterialLocalizations.of(context);
+    final Widget accountNameLine = accountName == null ? null : new DefaultTextStyle(
       style: theme.primaryTextTheme.body2,
       overflow: TextOverflow.ellipsis,
       child: accountName,
     );
-    Widget accountEmailLine = accountEmail == null ? null : new DefaultTextStyle(
+    final Widget accountEmailLine = accountEmail == null ? null : new DefaultTextStyle(
       style: theme.primaryTextTheme.body1,
       overflow: TextOverflow.ellipsis,
       child: accountEmail,
     );
-    if (onTap != null) {
-      if (accountEmailLine != null)
-        accountEmailLine = addDropdownIcon(accountEmailLine);
-      else
-        accountNameLine = addDropdownIcon(accountNameLine);
-    }
 
-    Widget accountDetails;
+    final List<Widget> rowChildren = <Widget>[];
+
     if (accountEmailLine != null || accountNameLine != null) {
-      accountDetails = new Padding(
-        padding: const EdgeInsets.symmetric(vertical: 8.0),
-        child: new Column(
-          crossAxisAlignment: CrossAxisAlignment.start,
-          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-          children: (accountEmailLine != null && accountNameLine != null)
-            ? <Widget>[accountNameLine, accountEmailLine]
-            : <Widget>[accountNameLine ?? accountEmailLine]
+      rowChildren.add(
+        new Expanded(
+          flex: 1,
+          child: new ExcludeSemantics(
+            excluding: false,
+            child: new Padding(
+              padding: const EdgeInsets.fromLTRB(16.0, 8.0, 0.0, 8.0),
+              child: new Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                children: (accountEmailLine != null && accountNameLine != null)
+                    ? <Widget>[accountNameLine, accountEmailLine]
+                    : <Widget>[accountNameLine ?? accountEmailLine],
+              ),
+            ),
+          ),
         ),
       );
     }
 
-    if (onTap != null)
-      accountDetails = new InkWell(onTap: onTap, child: accountDetails);
+    const double kAccountDetailsHeight = 56.0;
+
+    if (onTap != null) {
+      rowChildren.add(
+        new Semantics(
```

Comment:
Why do you need this container? I believe it will just produce an empty SemanticsNode?

Reply:
(see comment in test)

Reply:
I needed this initially so prevent it from merging the icon with the rest of the header. However, let me check again. It may not be necessary.

Reply:
Not necessary any more. Good catch. Removed.